src/Forms/FieldList.php

```diff
@@ -774,9 +729,9 @@ public function makeFieldReadonly($field)
      *
      * Please note that any tabs or other dataless fields will be clobbered by this operation.
      *
-     * @param array $fieldNames Field names can be given as an array, or just as a list of arguments.
+     * @param array<string>|string $fieldNames Field names can be given as an array, or just as a list of arguments.
      */
-    public function changeFieldOrder($fieldNames)
+    public function changeFieldOrder(array|string $fieldNames): void
     {
         // Field names can be given as an array, or just as a list of arguments.
         if (!is_array($fieldNames)) {
```

Comment:
Would be better to force using an array - though if you think otherwise I won't block merging for this.

Reply:
Leave as is for bc